ASP/ASP.sublime-syntax

```diff
@@ -2,10 +2,17 @@
 ---
 # http://www.sublimetext.com/docs/3/syntax.html
 name: ASP
+scope: source.asp
+
 file_extensions:
   - asa # asp is handled by HTML-ASP.sublime-syntax
   - vbs # Visual Basic Script
-scope: source.asp
+
+first_line_match: |-
+  (?xi:
+    ^ \s* \' \s* -\*- .*? \b(asp|vbs|vbscript)\b .*? -\*-  # editorconfig
```

Comment:
I'm thinking asp should perhaps be in the HTML-ASP sublime-syntax file instead? :)